addons/point_of_sale/static/src/js/models.js

```diff
@@ -1919,8 +1919,12 @@ exports.Paymentline = Backbone.Model.extend({
         this.amount = 0;
         this.selected = false;
         if (options.json) {
-            this.init_from_JSON(options.json);
-            return;
+            try {
+                this.init_from_JSON(options.json);
+                return;
+            } catch(error) {
+                this.pos.chrome.json_error({title: _t("Payment journal ID ") + json.statement_id +_t( " is not available in the point of sale.")});
```

Comment:
do not make this kind of string construction, it makes it very hard to translate, in js, use `_.str.sprintf`
see https://slides.com/mart-e/i18n-code#/4